docs/java/features/odata/generate-client.md

```diff
@@ -0,0 +1,270 @@
+---
+id: generate-java-odata-vmd-v2-v4
```

Comment:
Should we in general consider a prefix for language dependent docs, like `java-` and `js-` and make this part of a style guide? @artemkovalyov

Reply:
@marikaner  don't see a big deal in IDs naming as they are url-slags, that's it. SEO and Context wise it's better for words to follow a semantic structure of a sentence. The split happens on the structure level and the word `java` or `javascript` is still there.
If won't mind if js team would use some prefix still.

Reply:
By the way, the changed `id` has a type: `vmd` instead of `vdm`